ports/atmel-samd/boards/pycubed_mram/mpconfigboard.mk

```diff
@@ -0,0 +1,24 @@
+LD_FILE = boards/samd51x19-bootloader-external-flash.ld
```

Comment:
`LD_FILE` is no longer being used in `atmel-samd`. Instead, there is an `LD_TEMPLATE_FILE`, which gets filled in with various values. So you can remove this line from `mpconfigboard.mk` here and in the `pycubed` board file.